code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/dao/impl/ServiceExposeMapDaoImpl.java

```diff
@@ -78,7 +78,7 @@ public Instance createServiceInstance(Map<Object, Object> properties, Service se
             @Override
             public Object call() throws Exception {
                 objectProcessManager.scheduleStandardProcess(StandardProcess.CREATE, instance, null);
-                return null;
+                return instance;
```

Comment:
Since we don't do anything with the return value, I wonder should we change this to DeferredUtils.nest(new Runnable() { ...}) instead